src/CityDisplay.js

```diff
@@ -0,0 +1,89 @@
+import './App.css';
+import React from 'react';
+import axios from 'axios';
+
+class CityDisplay extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayCityData: false,
+      cityData: [],
+      userSearch: '',
+      errMsg: '',
+      errRender: false,
+
+    }
+  }
+
+  handleExplore = async () => {
+
+    try {
+      let cityResults = await axios.get('https://us1.locationiq.com/v1/search.php')
+
+      this.setState({
+        displayCityData: true,
+        cityData: cityResults.data.results
+      })
+    } catch (error) {
+      this.setState({
+        errRender: true,
+        errMsg: `An error has occurred: ${error.response.status}; ${error.response.data}`
+      })
+    }
+  }
+
+  handleCitySubmit = e => {
+    e.preventDefault();
+    let city = e.target.city.value
+    this.setState({
+      userSearch: e.target.city.value,
+    });
+    this.getCityData(city);
+  }
+
+  getCityData = async (city) => {
+    try {
+      let url = `https://us1.locationiq.com/v1/search.php?key=${process.env.local.REACT_APP_LOCATIONIQ_ACCESS_TOKEN}&q=${city}&format=json`;
+
+      let cityResults = await axios.get(url);
+
+      this.setState({
+        cityData: cityResults.data[0]
+      })
+    } catch (error) {
+      this.setState({
+        errRender: true,
+        errMsg: `An error has occurred: ${error.response.status}; ${error.response.data}`
```

Comment:
- line 56 is throwing an error, "cannot read properties of undefined reading 'status`;